src/Illuminate/Foundation/Console/AboutCommand.php

```diff
@@ -0,0 +1,154 @@
+<?php
+
+namespace Illuminate\Foundation\Console;
+
+use Illuminate\Console\Command;
+use Illuminate\Support\Composer;
+use Symfony\Component\Console\Attribute\AsCommand;
+
+#[AsCommand(name: 'about')]
+class AboutCommand extends Command
+{
+    /**
+     * The console command signature.
+     *
+     * @var string
+     */
+    protected $signature = 'about';
+
+    /**
+     * The name of the console command.
+     *
+     * This name is used to identify the command during lazy loading.
+     *
+     * @var string|null
+     *
+     * @deprecated
+     */
+    protected static $defaultName = 'about';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'About your application\'s environment';
+
+    /**
+     * The Composer instance.
+     *
+     * @var \Illuminate\Support\Composer
+     */
+    protected $composer;
+
+    /**
+     * The data to display.
+     *
+     * @var array
+     */
+    protected static $data = [];
+
+    /**
+     * Create a new command instance.
+     *
+     * @param  \Illuminate\Support\Composer  $composer
+     * @return void
+     */
+    public function __construct(Composer $composer)
+    {
+        parent::__construct();
+
+        $this->composer = $composer;
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return int
+     */
+    public function handle()
+    {
+        $this->gatherEnvironmentData();
+        $this->gatherDriverData();
+
+        collect(static::$data)->sortBy(function ($data, $key) {
+            $index = array_search($key, ['Environment', 'Drivers']);
+
+            if ($index === false) {
+                return 99;
+            }
+
+            return $index;
+        })
+        ->each(function ($data, $section) {
+            $this->newLine();
+            $this->components->twoColumnDetail('  <fg=green;options=bold>'.$section.'</>');
+
+            sort($data);
+
+            foreach ($data as $detail) {
+                [$label, $value] = $detail;
+
+                $this->components->twoColumnDetail($label, $value);
+            }
+        });
+
+        return 0;
+    }
+
+    /**
+     * Gather data about the application's environment.
+     *
+     * @return array
+     */
+    protected function gatherEnvironmentData()
+    {
+        static::add('Environment', [
+            'Laravel Version' => app()->version(),
```

Comment:
Why not `$this->laravel->version()`?

Reply:
I've switched to using this `$this->laravel` for consistency.